.github/workflows/docker.yml

```diff
@@ -75,6 +84,7 @@ jobs:
         uses: famedly/metadata-action@v4
         with:
           images: ${{ inputs.registry }}/${{ inputs.image_name }}
+          tags: ${{ inputs.tags }}
```

Comment:
I'm not sure how this magic works exactly, but I *think* we also need this in the docker metadata step in the merge job

Reply:
yep, I'm pretty sure you're right, I just forgot about the repeated metadata step..